package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "blip-cards",
-  "version": "2.7.4",
+  "version": "2.7.11",
```

Comment:
Why this is changing from 4 to 11?

Reply:
I'm trying to match the last release number from github. It seems that the last person forgot to update this package.json version number